examples/basic/index.html

```diff
@@ -58,6 +58,9 @@
       logtask1: { label: 'Test Log Task 1' },
```

Comment:
Minor point since this is just a test page, but might be nice to use action that less directly tie to what their bindings are to get people thinking about actions in the right way. Weather this is worth fixing mostly depends on whether this file servers more as an integration test or a usage example.

Reply:
Yep the current example is just used to test, so I'll create a more polished example with some text showing instructions and some kind of interactions like press X to change color of this cube or whatever